dockstore-cli-integration-testing/src/test/java/io/dockstore/common/CommonTestUtilities.java

```diff
@@ -149,8 +150,30 @@ private static void cleanStatePrivate1(DropwizardTestSupport<DockstoreWebservice
         Application<DockstoreWebserviceConfiguration> application = support.getApplication();
         application.run("db", "drop-all", "--confirm-delete-everything", configPath);
 
-        List<String> migrationList = Arrays.asList("1.3.0.generated", "1.3.1.consistency", "test.confidential1", "1.4.0", "1.5.0", "test.confidential1_1.5.0", "1.6.0", "1.7.0");
+        List<String> migrationList = Arrays.asList("1.3.0.generated", "1.3.1.consistency");
         runMigration(migrationList, application, configPath);
+
+        migrationList = Collections.singletonList("../dockstore-webservice/src/main/resources/migrations.test.confidential1.xml");
```

Comment:
dockstore-webservice isn't going to be present, is it? This may be covered in your notes already though.

Reply:
No notes yet, but in brief:
I left the infrastructure/tooling in place. 
The regular confidential bash tooling will leave the confidential scripts in this this location even though the Maven module will be gone. 

Ideally, the confidential testing bundle won't be needed for dockstore-cli
https://github.com/dockstore/dockstore/issues/2891